src/main/java/org/traccar/protocol/GatorProtocolDecoder.java

```diff
@@ -120,17 +175,102 @@ protected Object decode(
             position.set(Position.KEY_SATELLITES, flags & 0x0f);
 
             position.set(Position.KEY_STATUS, buf.readUnsignedByte());
-            position.set("key", buf.readUnsignedByte());
+            position.set(Position.KEY_IGNITION, buf.readUnsignedByte() == 0x01);
```

Comment:
Why are we changing this?

Reply:
It's clearly said in the protocol documents that this byte is regarding `ignition status`.
0x01 -> ON
0x00 -> OFF

The devices we have is also connected to ignition. I wonder why this protocol didn't use the already existing key.